From 9e60fcda0c29f5f4a01d6c1b21ac47af54a733e8 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 30 May 2013 16:05:25 +0300
Subject: [PATCH 059/228] be: remove kernel UT

Kernel UT will be added later. Currently they pollute the code base.
---
 Kbuild.in                      | 2 --
 Makefile.am                    | 2 --
 be/Kbuild.sub                  | 1 -
 be/ut/Kbuild.sub               | 1 -
 be/ut/seg.c                    | 1 -
 utils/linux_kernel/m0ut_main.c | 2 --
 6 files changed, 9 deletions(-)
 delete mode 100644 be/Kbuild.sub
 delete mode 100644 be/ut/Kbuild.sub

diff --git a/Kbuild.in b/Kbuild.in
index 2e89870..593eb91 100644
--- a/Kbuild.in
+++ b/Kbuild.in
@@ -59,7 +59,6 @@ include $(src)/mero/Kbuild.sub
 include $(src)/m0t1fs/Kbuild.sub
 
 include $(src)/addb/Kbuild.sub
-include $(src)/be/Kbuild.sub
 include $(src)/cob/Kbuild.sub
 include $(src)/conf/Kbuild.sub
 include $(src)/db/Kbuild.sub
@@ -105,7 +104,6 @@ utils/linux_kernel/m0ut-y = utils/linux_kernel/m0ut_main.o \
 m0ut_objects :=
 
 include $(src)/addb/ut/Kbuild.sub
-include $(src)/be/ut/Kbuild.sub
 include $(src)/ioservice/ut/Kbuild.sub
 include $(src)/layout/ut/Kbuild.sub
 include $(src)/lib/ut/Kbuild.sub
diff --git a/Makefile.am b/Makefile.am
index 3b04dcf..6378b22 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1025,8 +1025,6 @@ help:
 
 EXTRA_DIST += addb/Kbuild.sub \
               addb/ut/Kbuild.sub \
-              be/Kbuild.sub \
-              be/ut/Kbuild.sub \
               cob/Kbuild.sub \
               conf/Kbuild.sub \
               db/Kbuild.sub \
diff --git a/be/Kbuild.sub b/be/Kbuild.sub
deleted file mode 100644
index 119b726..0000000
--- a/be/Kbuild.sub
+++ /dev/null
@@ -1 +0,0 @@
-m0mero_objects += be/alloc.o be/seg.o
diff --git a/be/ut/Kbuild.sub b/be/ut/Kbuild.sub
deleted file mode 100644
index 12dda12..0000000
--- a/be/ut/Kbuild.sub
+++ /dev/null
@@ -1 +0,0 @@
-m0ut_objects += be/ut/seg.o
diff --git a/be/ut/seg.c b/be/ut/seg.c
index df3c35e..118bfc7 100644
--- a/be/ut/seg.c
+++ b/be/ut/seg.c
@@ -103,4 +103,3 @@ const struct m0_test_suite be_seg_ut = {
 		{ NULL, NULL }
 	}
 };
-M0_EXPORTED(be_seg_ut);
diff --git a/utils/linux_kernel/m0ut_main.c b/utils/linux_kernel/m0ut_main.c
index 6b30ea7..a3df329 100644
--- a/utils/linux_kernel/m0ut_main.c
+++ b/utils/linux_kernel/m0ut_main.c
@@ -33,7 +33,6 @@ MODULE_LICENSE("proprietary");
 /* sort test suites in alphabetic order */
 extern const struct m0_test_suite m0_klibm0_ut; /* test lib first */
 extern const struct m0_test_suite m0_addb_ut;
-extern const struct m0_test_suite be_seg_ut;
 extern const struct m0_test_suite buffer_pool_ut;
 extern const struct m0_test_suite bulkio_client_ut;
 extern const struct m0_test_suite m0_net_bulk_if_ut;
@@ -64,7 +63,6 @@ static void run_kernel_ut(int ignored)
 	/* sort test suites in alphabetic order */
 	m0_ut_add(&m0_klibm0_ut);  /* test lib first */
 	m0_ut_add(&m0_addb_ut);
-	m0_ut_add(&be_seg_ut);
 	m0_ut_add(&file_io_ut);
 	m0_ut_add(&buffer_pool_ut);
 	m0_ut_add(&bulkio_client_ut);
-- 
1.8.3.2

